src/backend/partaj/core/admin.py

```diff
@@ -292,6 +300,7 @@ class ReferralActivityAdmin(admin.ModelAdmin):
         "referral",
         "actor",
         "verb",
+        "message",
```

Comment:
IMO the message is not necessary to display in the table on the list view.

Reply:
Done